docs/components/modeler/desktop-modeler/element-templates/defining-templates.md

```diff
@@ -477,67 +339,16 @@ If a user removes the value in the configured control, it will also remove the m
 
 ##### Supported Bindings
 
-Camunda Platform 8 supports `optional` on the following binding types:
+The following binding types can be `optional`:
 
 - `zeebe:input`
 - `zeebe:output`
 - `zeebe:taskHeader`
 - `zeebe:property`
```

Comment:
I think we forgot about `bpmn:Message#zeebe:subscription#property` here (cf. https://github.com/bpmn-io/bpmn-js-element-templates/blob/main/src/cloud-element-templates/create/MessageZeebeSubscriptionBindingProvider.js#L39).

Reply:
Cf. https://github.com/camunda/camunda-platform-docs/pull/2369#discussion_r1283132249